docs/pages/docs/01-cloud-deployment/aws-deployment.md

```diff
@@ -72,7 +89,7 @@ To log in into one of the cluster nodes, simply use the ssh command:
 
 `ssh -i <private key file> ubuntu@<grakn instance DNS name or IP address>`
 
-#### Cluster health check
+#### <a name="cluster-check"></a> Cluster health check
```

Comment:
@kasper-piskorski what is this link, Kasper?

Reply:
markdown reference, allows you to jump to that section

Reply:
aren't these links automatically generated by the markdown-to-html generator @kasper-piskorski ? @lolski